entity-types/agent-infra/definition.yml

```diff
@@ -0,0 +1,5 @@
+domain: AGENT
+type: INFRA
```

Comment:
What is its role? 

How does it relate to the already existing infra host (that is both the agent and the underlying host)?

Reply:
my understanding is there is no entity representation for the infra agent itself. This will create an entity for the agent itself based on metadata sent to and used in FM. We can create future agent to host entity relationships(maybe monitors, etc.)